src/index.html

```diff
@@ -1,13 +1,57 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+
+  <body>
+    <section class="card" data-qa="card">
+      <div class="card__img-postion">
+        <img class="card__img"
+          src="./images/imac.jpeg"
+          alt="Apple A1419 iMac 27 diagonal Retina 5K Monoblock"
+        >
+      </div>
+
+      <h2 class="card__title">
+        APPLE A1419 iMac 27" Retina <br> 5K Monoblock (MNED2UA/A)
+      </h2>
+      <div class="card__id-text">Product code: 195434</div>
+
+
+      <div class="card__star-reviews-position">
+        <ul class="stars stars--4">
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+        </ul>
+
+        <span class="card__reviews">Reviews: 5</span>
+      </div>
+
+      <div class="card__price-position">
+        <span class="card__price-text">Price:</span>
+        <span class="card__price">$2,199</span>
+      </div>
+
+      <div
+        class="card__button-position"
+        href="/"
+        data-qa="hover"
+      >
+        <a class="card__button">
+          buy
+        </a>
+      </div>
```

Comment:
please check this part. it's better to leave the `a` element only
```suggestion
      <a
        class="card__button"
        href="/"
        data-qa="hover"
      >
        buy
      </div>
```